plugins/cuda/cuda_plugin.c

```diff
@@ -460,8 +461,15 @@ CR_PLUGIN_REGISTER_HOOK(CR_PLUGIN_HOOK__RESUME_DEVICES_LATE, cuda_plugin_resume_
 
 int cuda_plugin_init(int stage)
 {
-	int ret = cuda_checkpoint_supports_flag("--action");
+	int ret;
 
+	if (!fault_injected(FI_PLUGIN_CUDA_FORCE_ENABLE) && access("/dev/nvidiactl", F_OK)) {
```

Comment:
@avagin The [gpu-operator](https://github.com/NVIDIA/gpu-operator) for Kubernetes uses [`/run/nvidia/driver`](https://github.com/NVIDIA/gpu-operator/blob/d38a36f6b5d7a1e0dd6da600be4e061c7c83a183/controllers/object_controls.go#L175) as default path for driver installation.

In this case, `/dev/nvidiactl` does not exist:
```
# ls -al /dev/nvidiactl
ls: cannot access '/dev/nvidiactl': No such file or directory
# ls -al /run/nvidia/driver/dev/nvidiactl
crw-rw-rw- 1 root root 195, 255 Nov  1 15:45 /run/nvidia/driver/dev/nvidiactl
```